### ISSUE TITLE
Lycanites mobs abilities impacts thirst.

### ISSUE THREAD

**Issue by Markov82** [Charles445/SimpleDifficulty#2]
Hi !
Using the creature abilities of the Lycanite Mobs mod impacts the level of thirst, drastically. Would it be possible to add a setting to the config file to disable or reduce this? 
Thank you !

**Comment by Charles445** [comment]
I'll have to address it directly at some point...

For now, check the mod config, in the server, thirst category
Set "ThirstAttacking" to 0
See if that does it!

**Comment by Markov82** [comment]
Yes it works but suddenly, attacking normally (even with a sword for example) no longer consumes thirst.
